server/src/main/java/com/objectcomputing/checkins/security/LocalUserPasswordAuthProvider.java

```diff
@@ -36,7 +36,7 @@ public LocalUserPasswordAuthProvider(CurrentUserServices currentUserServices,
     @Override
     public Publisher<AuthenticationResponse> authenticate(@Nullable HttpRequest<?> httpRequest, AuthenticationRequest<?, ?> authReq) {
         String email = authReq.getIdentity().toString();
-        MemberProfile memberProfile = currentUserServices.findOrSaveUser(email, email);
+        MemberProfile memberProfile = currentUserServices.findOrSaveUser(email, email, email);
```

Comment:
Please see my comment for `LocalLoginController.java`. The input parameter `authReq` only contains username and password, which are the email and role, respectively, passed by the front-end. We can't get the first name/last name here as far as I understand.

Reply:
If I'm wrong about the above, then you are correct. If not, I think this will need to change.

You can validate it by trying to login locally with my email address or Holly's.

Reply:
The existing member profiles are loaded correctly. Only new member profiles with non-existing emails will have their first and last names equal to their emails.